go/enclave/rollupchain/rollup_chain.go

```diff
@@ -168,22 +116,26 @@ func (rc *RollupChain) UpdateL2Chain(batch *core.Batch) (*common.BatchHeader, er
 	rc.blockProcessingMutex.Lock()
 	defer rc.blockProcessingMutex.Unlock()
 
-	// We retrieve the genesis batch from the L1 chain instead.
-	if batch.Header.Number.Cmp(big.NewInt(int64(common.L2GenesisHeight))) == 0 {
```

Comment:
We no longer expect to retrieve the genesis from a rollup as a validator. We expect to receive it as a batch instead.